packages/dd-trace/test/ci-visibility/exporters/git/git_metadata.spec.js

```diff
@@ -220,28 +220,29 @@ describe('git_metadata', () => {
   })
   describe('validateGitRepositoryUrl', () => {
     it('should return false if Git repository URL is invalid', () => {
-      const invalidUrl1 = 'https://test.com'
-      const invalidUrl2 = 'https://test.com'
-      const invalidUrl3 = 'http://test.com/repo/dummy.4git'
-      const invalidUrl4 = 'https://test.com/repo/dummy.gi'
-      const invalidUrl5 = 'www.test.com/repo/dummy.git'
-      const invalidUrl6 = 'test.com/repo/dummy.git'
-
-      const invalidUrls = [invalidUrl1, invalidUrl2, invalidUrl3, invalidUrl4, invalidUrl5, invalidUrl6]
+      const invalidUrls = [
+        'www.test.com/repo/dummy.git',
+        'test.com/repo/dummy.git',
+        'test.com/repo/dummy'
+      ]
       invalidUrls.forEach((invalidUrl) => {
-        expect(validateGitRepositoryUrl(invalidUrl)).to.be.false
+        expect(validateGitRepositoryUrl(invalidUrl), `${invalidUrl} is a valid URL`).to.be.false
       })
     })
     it('should return true if Git repository URL is valid', () => {
-      const validUrl1 = 'https://test.com/repo/dummy.git'
-      const validUrl2 = 'http://test.com/repo/dummy.git'
-      const validUrl3 = 'https://github.com/DataDog/dd-trace-js.git'
-      const validUrl4 = 'git@github.com:DataDog/dd-trace-js.git'
-      const validUrl5 = 'git@github.com:user/repo.git'
+      const validUrls = [
+        'https://test.com',
```

Comment:
for me this sounds weird, `https://test.com` as valid git url, without any `/path` at the end.

Reply:
sadly I think it's technically possible for you to have such a repository URL. There are no good authoritative sources for such urls (that I found anyway)

Reply:
if you find any, I'd be more than happy to change this 😄